README.md

```diff
@@ -1,58 +1,55 @@
+6. **Locked down envs**: Downloading a VSIX from the marketplace allows developers who are unable to freely install software to install extensions manually.
+7. **Missing dependencies**: Users may run into situations where .NET Core cannot run as-is, requiring the installation of missing pieces.
 
-1. **Duplication of .NET Core + slow updates**: The way things are going at the moment, each extension is acquiring its own copy of .NET core at ~30mb each. The binary contents are dropped inside the extensions folder in VS Code so that when the extension is uninstalled, the copy of .NET Core goes away too. This also forces .NET core to be downloaded on every extension version update.
-1. **Clean up**: To avoid slow updates extensions could consider installing .NET Core in a non-VSCode folder location; however, in this case, there's no great mechanism to tell when all extensions that use .NET Core are uninstalled. So, .NET Core would likely need to be left behind. 
-1. **Servicing / floating versions**: With newer releases of various OS flavors and .NET Core releases there are occasions when incompatibilities arise. With the existing infrastructure every team that re-ships .NET Core needs to re-ship their extension, re-dynamically acquire .NET and then cross their fingers. This can typically be avoided by dynamically acquiring the latest flavor of .NET Core (if the extension wants) to avoid re-shipping every .NET Core extension.
-1. **Corruption due to lack of control**: VS Code can be shut down mid-download or unzip which can result in corrupted bits.  After a year of heavy use and attempting to resolve these problems, Live Share is still seeing corruption in some cases. Live Share doesn't have control over the process lifecycle which makes solving this difficult.
-1. **Barrier to entry due to network security policies**: There have been a number of companies that "whitelist" sites instead of blacklisting them for security. As a result, developers have to request access to URIs to enable access. The VS marketplace is typically already unblocked if they are VS Code users while others may not be. 
-1. **Locked down envs**: There have been situations where developers are not allowed to freely download software. The ability to download a VSIX from the marketplace allows these customers to centrally acquire extensions and install them manually. Any dynamic acquisition experience does not easily work in these environments.
-1. **.NET Core missing dependencies**: VSCode runs on more platforms than .NET Core and therefore users run into situations where .NET Core cannot run as-is (generic containers, Alpine etc.) once dynamically acquired. This leads extension authors to detect these situations and attempt to prompt users to install missing pieces.
+This extension attempts to solve the above issues.
 
-This repo attempts to solve the above issues.
+## Contributing to Repository
```

Comment:
Does this appear in the gallery or just in the repo?

Reply:
It appears in both places. It seems to be the norm for gallery listings to have this kind of information ([example](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions)).